include/os/macos/spl/sys/Makefile.am

```diff
@@ -0,0 +1,5 @@
+KERNEL_H = \
+	$(top_srcdir)/include/os/macos/spl/sys/atomic.h \
+	$(top_srcdir)/include/os/macos/spl/sys/condvar.h \
+	$(top_srcdir)/include/os/macos/spl/sys/mutex.h \
+	$(top_srcdir)/include/os/macos/spl/sys/types.h
```

Comment:
Why does linux define `KERNEL_H`? Is it used for something

Reply:
The headers were split in to `KERNEL_H`, `USER_H` and `COMMON_H` to allow installing the user and kernel devel headers in the right locations.  On the Linux side there are other kernel modules (Lustre) which need to build against ZFS, and in user space there are a couple libzfs and libzfs_core consumers.